localgov_microsites_base.theme

```diff
@@ -8,10 +10,17 @@
  * Implements hook_preprocess_HOOK().
  */
 function localgov_microsites_base_preprocess_html(&$variables) {
-  $route = \Drupal::request()->attributes->get('_route');
 
-  if ($route === 'entity.group.canonical' || $route === 'entity.group_content.canonical') {
+  $parent_group_id = \Drupal::service('class_resolver')
+  ->getInstanceFromDefinition(DomainGroupHelper::class)
+    ->getActiveDomainGroup();
+
+    if (empty($parent_group_id)) {
     $parent_group_id = \Drupal::request()->attributes->get('group')->get('id')->value;
```

Comment:
This code was safe because it was checking `$route === 'entity.group.canonical' || $route === 'entity.group_content.canonical')` first.

It would probably make as much sense to just check `if (!empty(\Drupal::request()->get('group')`

Reply:
```
  $parent_group_id = \Drupal::service('class_resolver')
    ->getInstanceFromDefinition(DomainGroupHelper::class)
    ->getActiveDomainGroup();
  $group = \Drupal::entityTypeManager()->getStorage('group')->load($parent_group_id);

  if (empty($group)) {
    $group = \Drupal::request()->attributes->get('group');
  }

  if (!empty($group)) {

  .... all the rest
```